lib/admin/WP_Auth0_Admin_Appearance.php

```diff
@@ -290,10 +290,11 @@ public function render_extra_conf( $args = [] ) {
 	 * @return array
 	 */
 	public function basic_validation( $old_options, $input ) {
-		$input['form_title']    = empty( $input['form_title'] ) ? '' : sanitize_text_field( $input['form_title'] );
-		$input['icon_url']      = empty( $input['icon_url'] ) ? '' : esc_url( $input['icon_url'], [ 'http', 'https' ] );
-		$input['gravatar']      = empty( $input['gravatar'] ) ? 0 : 1;
-		$input['primary_color'] = empty( $input['primary_color'] ) ? '' : sanitize_text_field( $input['primary_color'] );
+		$input['passwordless_enabled'] = ( isset( $input['passwordless_enabled'] ) ? $input['passwordless_enabled'] : 0 ) == 1;
```

Comment:
Just a small suggestion. Might add readability to extract a constant for 0 and 1 like `OPTION_DISABLED`, `OPTION_ENABLED`

Reply:
Well ... it's not as simple as that. "Disabled" and "enabled" depends on context. The UI sends `"1"` as enabled, which should be converted to `true`.

By the end of this series of PRs, we'll be normalized on `true/false` for checking.